docs/source/recipes/plot_13_recipe.py

```diff
@@ -0,0 +1,265 @@
+    alpha=0.3,
+    transform=ccrs.PlateCarree(),
+)
+cfp.plotvars.mymap.add_patch(rectangle)
+cfp.plotvars.mymap.text(
+    -120,
+    -3,
+    "3",
+    horizontalalignment="center",
+    fontsize=14,
+    weight="bold",
+    transform=ccrs.PlateCarree(),
+)
+
+# Niño 4 region (5N-5S, 160E-150W):
+rectangle = mpatches.Rectangle(
+    (-200, -5),
+    50,
+    10,
+    hatch="oo",
+    fill=False,
+    linewidth=1,
+    edgecolor="black",
+    alpha=0.3,
+    transform=ccrs.PlateCarree(),
+)
+cfp.plotvars.mymap.add_patch(rectangle)
+cfp.plotvars.mymap.text(
+    -175,
+    -3,
+    "4",
+    horizontalalignment="center",
+    fontsize=14,
+    weight="bold",
+    transform=ccrs.PlateCarree(),
+)
+cfp.gclose()
+
+# %%
+# 6. Calculate the Niño 3.4 index and standardize it to create an anomaly index.
+# The `collapse <https://ncas-cms.github.io/cf-python/method/cf.Field.collapse.html>`_
+# method is used to calculate the mean over the longitude (X) and latitude (Y)
+# dimensions. The result, ``nino34_index``, represents the average SST in the
+# defined Niño 3.4 region for each time step. In the variable ``base_period``,
+# ``nino34_index`` is subset to only include data from the years 1961 to 1990.
+# This period is often used as a reference period for calculating anomalies.
+# The variables ``climatology`` and ``std_dev`` include the mean and the
+# standard deviation over the time (T) dimension of the ``base_period`` data
+# respectively. The line for varaible ``nino34_anomaly`` calculates the
+# standardized anomaly for each time step in the ``nino34_index`` data. It
+# subtracts the ``climatology`` from the ``nino34_index`` and then divides by
+# the ``std_dev``. The resulting ``nino34_anomaly`` data represents how much
+# the SST in the Niño 3.4 region deviates from the 1961-1990 average, in units
+# of standard deviations. This is a common way to quantify climate anomalies
+# like El Niño and La Niña events:
+nino34_index = region.collapse("X: Y: mean")
+
+# %%
+
+base_period = nino34_index.subspace(T=cf.year(cf.wi(1961, 1990)))
+climatology = base_period.collapse("T: mean")
+std_dev = base_period.collapse("T: sd")
+
+# %%
+
+nino34_anomaly = (nino34_index - climatology) / std_dev
+
+# %%
+# 7. A moving average of the ``nino34_anomaly`` along the time axis, with a
+# window size of 5 (i.e. an approximately 5-month moving average) is calculated
+# using the
+# `moving_window <https://ncas-cms.github.io/cf-python/method/cf.Field.moving_window.html>`_
+# method. The ``mode='nearest'`` parameter is used to specify how to pad the
+# data outside of the time range. The resulting ``nino34_rolling`` variable
+# represents a smoothed version of the ``nino34_anomaly`` data. It removes
+# short-term fluctuations and highlights longer-term trends or cycles.
+nino34_rolling = nino34_anomaly.moving_window(
+    method="mean", window_size=5, axis="T", mode="nearest"
+)
+
+# %%
+# 8. Define El Niño and La Niña events by creating boolean masks to identify
```

Comment:
```suggestion
# 8. Define El Niño and La Niña events by creating Boolean masks to identify
```

(It tends to be capitalised everywhere since it comes from the name Boole after the mathematician, in case you weren't aware of the context.)